cmd/snap/cmd_changes.go

```diff
@@ -70,11 +70,12 @@ func (c *cmdChanges) Execute(args []string) error {
 	}
 
 	if allDigits(c.Positional.Snap) {
-		return fmt.Errorf(`%s changes command expects a snap name, try: %[1]s change %s`, os.Args[0], c.Positional.Snap)
+		// TRANSLATORS: the first %s, and %[1]s, is the argv[0] (typically "snap"); the last %s is the argument given by the user to 'changes'
+		return fmt.Errorf(i18n.G(`%s changes command expects a snap name, try: %[1]s change %s`), os.Args[0], c.Positional.Snap)
```

Comment:
do we actually want this (in which case several other places need it), or should we just use "snap"?